devbot/database.py

```diff
@@ -51,4 +51,13 @@ class XKCD(SQLAlchemyBase):
     year = sa.Column(sa.String(4))
 
 
+class Doodle(SQLAlchemyBase):
+    """ Representation of a Doodle poll. """
+
+    __tablename__ = "doodle"
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
```

Comment:
Using id shadows a function, this might cause issues. Check the [docs](https://docs.python.org/3/library/functions.html#id) for more info.

Reply:
Good point